src/common/translations/es.json

```diff
@@ -94,7 +94,7 @@
       "recoveryPhrase": {
         "title": "Recovery Phrase",
         "note1": "Escriba o copie estas palabras",
-        "note2": "en el orden correcto y guárdelos",
+        "note2": "en el orden correcto y guárdalos",
```

Comment:
The change should be from "guárdelos" to "guárdelas"

Reply:
okay, I'll update it. My chrome suggested the one I added.

Reply:
Thanks! fixes.